### PR TITLE
New Command System

### PR DESCRIPTION
**Pull request by creatorfromhell**
- Added a new command system
- Added a help command
- Added a command event

### CODE REVIEW

**Comment by SirCmpwn** [comment]
This looks good, with the exception of the transfer of responsibility over command recognition to the TrueCraft.Server.MultiplayerServer.The idea of `/` as a command prefix is unique to the implementation and should be handled in Program.cs. It's not unlikely that we'll eventually split the frontend out into its own project.

**Comment by creatorfromhell** [comment]
I removed the command event code.

**Comment by SirCmpwn** [comment]
Great!

File: TrueCraft/Commands/CommandManager.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    {
+        public IList<ICommand> Commands { get; set; }
+
+        public CommandManager()
+        {
+            Commands = new List<ICommand>();
+            LoadCommands();
+        }
+
+        private void LoadCommands() {
```
Comment (SirCmpwn) [inline TrueCraft/Commands/CommandManager.cs]: Style error here

File: TrueCraft/Program.cs [hunk 2]
```diff
@@ -9,11 +9,13 @@
 using TrueCraft.API;
 using TrueCraft.Core.Windows;
 using System.IO;
+using TrueCraft.Commands;
 
 namespace TrueCraft
 {
     class MainClass
     {
+        public static CommandManager CManager;
```
Comment (SirCmpwn) [inline TrueCraft/Program.cs]: Can you name this "CommandManager" instead?

**Comment by creatorfromhell** [comment]
Sorry about the style error. I didn't notice it before I pushed the commit.

**Comment by SirCmpwn** [comment]
Thanks!

**Pull request merged**